supermarq-benchmarks/supermarq/benchmarks/bit_code.py

```diff
@@ -20,7 +20,12 @@ class BitCode(Benchmark):
 
     def __init__(self, num_data_qubits: int, num_rounds: int, bit_state: List[int]) -> None:
```

Comment:
Likewise from earlier comment: 
```suggestion
    def __init__(self, num_data_qubits: int, num_rounds: int, bit_state: typing.Sized) -> None:
```